mdata/aggmetric.go

```diff
@@ -66,6 +66,12 @@ func NewAggMetric(store Store, cachePusher cache.CachePusher, key string, retent
 	return &m
 }
 
+func (a *AggMetric) maybeSyncChunkSaveState(ts uint32) {
```

Comment:
I went with the maybe* because:
1) I thought we could establish that as a convention within metrictank (that maybeX means execute X if the receiver is not nil)
2) I've used the same convention in other projects/languages
3) we can also clarify the behavior in a doc comment (but that still doesn't make it obvious when reading the code from the call site)

But I don't care that much. And making it more explicit at the expense of a few sloc works for me, so i pushed that change.